apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -293,6 +298,14 @@ def version
       @ood_app.version
     end
 
+    # To allow the configuration of BatchConnect apps that will not show in the navigation menu
+    # This is useful for the creation of multiple preset pinned apps
+    # for the same application with different values
+    # @return [boolean] if this app should appear in the navigation
+    def show_in_nav?
+      form_config.fetch(:show_in_nav, true)
+    end
```

Comment:
This is the piece I've got to think on.  With all the rules & configurations available to show/hide apps in the navbar, it just seems like there are like 5 ways to do a thing. Very hard to document & convey to system admins.

I guess I'm just longing for something simpler for admins, and us for that matter.

Old/default logic already has this
https://github.com/OSC/ondemand/blob/5a3008cbba39c5c264d240f7a681e6114c0cbf1e/apps/dashboard/app/apps/ood_app.rb#L33-L35

That said - IDK if there's anything we can do at this moment. It's a larger issue than this pull request for sure.

Reply:
I tried to make it work with `should_appear_in_nav?`, but I had problems with `BatchConnect` sub apps.
sub apps only provide one file for the config, and is loaded into the `form_config` object.

Reply:
I just wonder if we want to rethink this before we pull it in. Can you pull this specific feature out of this PR so I can evaluate it on it's own?

I feel like as-is it'll need re-worked, so I'd rather just defer it altogether.

Reply:
OK - will pull the `show_in_nav` feature out. We can discuss ideas in our next meeting.

Reply:
You can submit it as a separate PR - I just want to think it through for a second.

Reply:
`show_in_nav` is out. Will create a separate PR later on.

Reply:
Thanks! I know it's a bit of a pain, it's just so many moving parts we have to be smart about adding more parts.